components/Layout.js

```diff
@@ -37,7 +37,7 @@ const Footer = styled.footer`
 
 const Copyright = styled.p`
   margin: 0;
-  padding-right: 7.5%;
+  padding-right: 7.5vw;
```

Comment:
I did it with "vw" because of H1 `width: 85vw`. I want to do padding-left of copyright equal to padding of `border-bottom` of H1-Title (component TitleBar / Headline).